pkg/rancher-desktop/components/UpdateStatus.vue

```diff
@@ -178,7 +178,11 @@ class UpdateStatus extends UpdateStatusProps {
     if (typeof markdown !== 'string') {
       return undefined;
     }
-    const unsanitized = marked(markdown);
+    // Here's the explanation of the following unorthodox typecast:
+    // The signature of `marked.marked` is, with version 11:
+    // marked(src: string, options?: MarkedOptions): string | Promise<string>
+    // It returns a Promise<string> if `options.async` is true, otherwise a string.
```

Comment:
They have it this way because arbitrary extensions can cause this to be async; it works because we're not using any of the extensions. But close enough.

(They have a separate overload for `marked(src, options?: {async: true} & MarkedOptions): Promise<string>` that covers the explicitly-async case.)